src/framework/components/anim/system.js

```diff
@@ -59,7 +59,11 @@ class AnimComponentSystem extends ComponentSystem {
         if (data.masks) {
             Object.keys(data.masks).forEach((key) => {
                 if (component.layers[key]) {
-                    component.layers[key].assignMask(data.masks[key].mask);
+                    const mask = {};
+                    Object.keys(data.masks[key].mask).forEach((maskKey) => {
```

Comment:
it might be worth caching `data.masks[key].mask` in a variable since it's used in a loop below, but since this is initialisation code might not be worth it.

Reply:
👍 done